inference/codegen.py

```diff
@@ -2,6 +2,8 @@
 
 _model = LocalHuggingfaceModel(
     "Salesforce/codegen-16B-multi",
-    dict(low_cpu_mem_usage=True))
+    dict(low_cpu_mem_usage=True),
+    set_pad_token_id_plus_2=True,
```

Comment:
Could there be a comment explaining why is this option necessary?

Reply:
sure. it is needed to suppress a warning that is significant